src/panels/media-browser/ha-bar-media-player.ts

```diff
@@ -260,7 +265,7 @@ class BarMediaPlayer extends LitElement {
 
   protected updated(changedProps: PropertyValues) {
     super.updated(changedProps);
-    if (!this.hass || !changedProps.has("hass")) {
+    if (!this.hass || (!changedProps.has("hass") && !this._browserPlayer)) {
```

Comment:
Are you sure about this @zsarnett ? We still need the interval even with a browser media player